packages/autocomplete/src/examples/multiselect.md

```diff
@@ -0,0 +1,304 @@
+The `MultiSelect` layout operates similarly to the `Autocomplete`, but must
+manage the interaction between the Input and it's collection of Tags.
+
+This examples includes an example of "copy-to-clipboard" functionality for tags.
```

Comment:
grammar ;)

Reply:
Grammar is fixed. I also rewrote the intro to be more clear.